config/lint.xml

```diff
@@ -17,6 +17,7 @@
     <issue id="UselessParent" severity="error" />
     <issue id="BinaryOperationInTimber" severity="error" />
     <issue id="RelativeOverlap" severity="error" />
+    <issue id="Typos" severity="error" />
```

Comment:
Is this potentially risky? I'm worried there might be colloquialisms etc that show up as typos from the translators.

Reply:
You might be right, if that happen then we can turn this rule off but for now I find it helpful because it will help us fix bugs in new translations.

Reply:
Good point. It's really easy to disable if we run into problems.